### ISSUE TITLE
New Issue Label system

### ISSUE THREAD

**Issue by igreulich** [isaacs/github#85]
Has anyone used the new issue label system, yet? It is far to much clicking to manage labels.

Currently you click the checkbox of the issue you wish to modify the labels on. Then you click the label button; the label menu drops down. Then you select the labels you wish to add. Then you click the apply button. Then the label menu closes.

Now if you also want to remove labels, you get to do all of that over again. Like I said, to much clicking.

How it should work;

Click the checkbox of the issue you would like to modify the labels on. Then click the label button; the label menu opens. The labels the issue has should be highlighted already. Add the labels you want, click the little x to remove those you want. Click apply. The label menu closes.

Boom! Done. 1/2 the clicks. (If I had a mic, I would drop it... :)

**Comment by TPS** [comment]
@igreulich Did you close this because it's resolved, or something else?

**Comment by igreulich** [comment]
I suppose I could have commented as much;

  

I closed it because, the issues feature/section has been completely overhauled
since I created the issue.

  

I am reevaluating the labels, but I use them day in and day out now, and I
can't think of any issues I have.

  
\--

Ian Greulich

www.greulich.me

  

On Apr 22 2016, at 10:51 pm, TPS &lt;notifications@github.com&gt; wrote:  

> [@igreulich](https://github.com/igreulich) Did you close this because it's
resolved, or something else?

>

> —  
You are receiving this because you were mentioned.  
Reply to this email directly or [view it on GitHub](https://github.com/isaacs/
github/issues/85#issuecomment-213655316)![](https://github.com/notifications/b
eacon/AAIznzEQnHLN3mlk0EA05lCnc9cEe6u_ks5p6YkmgaJpZM4A5-sf.gif)
